core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -401,6 +402,9 @@ export class LoggedInUser extends BaseUser {
     await this.page.$eval(weCannotContentId, element =>
       element.getElementsByTagName('a')[0].click()
     );
+    await this.page.waitForNavigation({waitUntil: ['load', 'networkidle2']});
+    await this.page.waitForSelector(_420millionPageHeaderSelector);
+
     if (this.page.url() !== _420MillionUrl) {
       throw new Error('The 420 Million link does not open the right page!');
```

Comment:
I suggest printing the incorrect URL here in this error message (and in other similar error messages) -- it would help with debugging.

Reply:
Done.